core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3067,10 +3076,12 @@ private void scheduleAsyncPersistenceInternal(LockedInodePath inodePath) throws
    * @param inodePath the Alluxio inode path to sync with UFS
    * @param lockingScheme the locking scheme used to lock the inode path
    * @param syncDescendantType how to sync descendants
+   * @param blockDeleter the block deleter
    * @return true if the sync was performed successfully, false otherwise (including errors)
    */
   private boolean syncMetadata(JournalContext journalContext, LockedInodePath inodePath,
-      LockingScheme lockingScheme, DescendantType syncDescendantType) {
+      LockingScheme lockingScheme, DescendantType syncDescendantType,
+      BlockDeletionContext blockDeleter) {
```

Comment:
NIT: move block context parameter next to journal context parameter (or vice versa)